src/common/transformations/include/transformations/common_optimizations/optimize_gather_nd.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+#include <transformations_visibility.hpp>
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API OptimizerGatherND;
```

Comment:
The transformation purpose is to decompose GatherND.
What do you think about aligning the name with other decompositions and moving the files to the [src/common/transformations/include/transformations/op_conversions/](https://github.com/openvinotoolkit/openvino/tree/b98e5c7256015518f8ff71a0a590e2ef5478c1dd/src/common/transformations/include/transformations/op_conversions/) directory?